src/main/mod/data/console/commands.csv

```diff
@@ -13,6 +13,7 @@ AddOrdnancePoints,org.lazywizard.console.commands.AddOrdnancePoints,"core,campai
 AddShip,org.lazywizard.console.commands.AddShip,"core,campaign",addship <variantID> [optionalAmount],"Tries to create a ship with the supplied variant ID and adds it to your fleet. This command is case-sensitive, but it will try with different capitalization if it fails. If an amount is given, it will spawn that many ships of that ID in your fleet. Ensure you have the required supplies!\nA ship name with no variant attached will generate an empty hull (the same result as 'addship <shipname>_Hull').\nSupports reversed arguments."
 AddSkillPoints,org.lazywizard.console.commands.AddSkillPoints,"core,campaign",addskillpoints <amount>,"Adds the specified amount of skill points to your character."
 AddSpecial,org.lazywizard.console.commands.AddSpecial,"core,campaign",addspecial <specialId> [optionalData],"Adds a special item to your fleet's cargo with the given data, or null data if no dara argument is passed in.\nNote that this will cause a game crash if no or invalid data is passed into an object that requires it!"
+AddSubmarket,org.lazywizard.console.commands.AddSubmarket,"core,market",addsubmarket <submarketId> [optionalParams],"Adds a submarket to a market.\nUse 'list submarkets' to list all valid submarkets."
```

Comment:
```suggestion
AddSubmarket,org.lazywizard.console.commands.AddSubmarket,"core,market",addsubmarket <submarketId>,"Adds a submarket to a market.\nUse 'list submarkets' to list all valid submarkets."
```

AddSubmarket doesn't take an optional parameter. Other than that everything looks good!